<launch>
    <!-- <node type="rviz" name="rviz" pkg="rviz" args="-d /home/koceti/catkin_ws/rviz_config/adus_path_read.rviz"/> -->

    <node pkg="adus_cto" type="key_input_node" name="key_input_node"/>

    <node pkg="adus_cto" type="path_read_node" name="path_read_node" output="screen">
        <param name="frame_id" value="map"/>
        <param name="path_01" value="/home/adus/koceti_hil_ws/src/adus_cto/path/rddf_path_lla/231122061500_route.txt"/>
        <param name="path_02" value="/home/koceti/catkin_ws/path/origin_trajectory_lla/241031120219_route.txt"/>
        <param name="path_03" value="/home/koceti/catkin_ws/path/rddf_path_utm/241031120219_route.txt"/>
    </node>

</launch>
